profiler/README.md

```diff
@@ -1,9 +1,46 @@
 
-The profiler is in its infancy and is disabled by default.
+🚨 The profiler is in beta and its use is currently restricted.
```

Comment:
What does it mean "restricted"?

Reply:
It means that users shouldn't have an expectation yet that it just works and that they need to be working directly with us in order to use the thing in a meaningful way.

Reply:
Anyway, it's been changed to experimental.